tests/gui/models/store.spec.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const m = require('mochainon');
+const Store = require('../../../lib/gui/models/store');
+
+describe('Browser: Store', function() {
+
+  describe('given a SET_FLASH_STATE action', function() {
+
+    it('should not allow percentages over 100', function() {
+      const OVER_HUNDRED = 101;
```

Comment:
IIRC the eslint rules are different for the `tests/` directory, so you don't need to create constants-for-numbers :-)